### PR TITLE
test: 11 tests from others

### PR DESCRIPTION
**Pull request by mintbomb27**
## Related Issue(s)
#546 

## Description
Contains 11 kuttl tests from others

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [ ] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: other/res/restrict-node-label-changes/03-check.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
```
Comment (chipzoller) [inline other/res/restrict-node-label-changes/03-check.yaml:4]: I would also like to see an addition, modification, and removal of another label aside from `foo`.

File: other/res/restrict-node-label-creation/03-check.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
```
Comment (chipzoller) [inline other/res/restrict-node-label-creation/03-check.yaml:4]: Same sort of test here. Make sure you can set some other label aside from `foo`.

File: other/res/restrict-pod-controller-serviceaccount-updates/03-manifests.yaml
Comment (chipzoller) [inline other/res/restrict-pod-controller-serviceaccount-updates/03-manifests.yaml:1]: How about some negative tests ensuring that another type of update to the `spec` object for a Deployment and a CronJob are allowed so long as it's not in the `serviceAccountName` field?

File: other/res/restrict-secret-role-verbs/cr-bad.yaml
Comment (chipzoller) [inline other/res/restrict-secret-role-verbs/cr-bad.yaml:1]: Let's also try another "bad" resource in which only one of the forbidden verbs appears. That's what this policy checks.

File: other/res/restrict-secret-role-verbs/role-bad.yaml
Comment (chipzoller) [inline other/res/restrict-secret-role-verbs/role-bad.yaml:1]: Ditto here.

File: other/res/restrict-secrets-by-label/secret.yaml
Comment (chipzoller) [inline other/res/restrict-secrets-by-label/secret.yaml:1]: Don't see a Secret here with a different label than `status` and no labels at all.

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: other/res/restrict-node-label-changes/03-check.yaml (hunk 1 above)
Comment (mintbomb27) [reply other/res/restrict-node-label-changes/03-check.yaml:4]: Okayy, added.

File: other/res/restrict-node-label-creation/03-check.yaml (hunk 2 above)
Comment (mintbomb27) [reply other/res/restrict-node-label-creation/03-check.yaml:4]: Added.

File: other/res/restrict-pod-controller-serviceaccount-updates/03-manifests.yaml
Comment (mintbomb27) [reply other/res/restrict-pod-controller-serviceaccount-updates/03-manifests.yaml:1]: Okay added.

File: other/res/restrict-secret-role-verbs/cr-bad.yaml
Comment (mintbomb27) [reply other/res/restrict-secret-role-verbs/cr-bad.yaml:1]: Added.

File: other/res/restrict-secrets-by-label/secret.yaml
Comment (mintbomb27) [reply other/res/restrict-secrets-by-label/secret.yaml:1]: Should have added that, sorry! Added now.

**Review by chipzoller — APPROVED** [review]
Nice!

**Pull request merged**